src/expertsystem/reaction/__init__.py

```diff
@@ -278,7 +278,17 @@ def _determine_graph_settings(
         # pylint: disable=too-many-locals
         final_state_edges = graph.get_final_state_edges()
         initial_state_edges = graph.get_initial_state_edges()
-        graph_settings: List[GraphSettings] = [GraphSettings({}, {})]
+        graph_settings: List[GraphSettings] = [
+            GraphSettings(
+                edge_settings={
+                    edge_id: self.interaction_type_settings[
+                        InteractionTypes.Weak
```

Comment:
Why weak by default?

Reply:
That's indeed quite misleading... The thing is that there are interaction rules, which belong to an interaction and must involve edges going into AND out of a node. The other part of rules are just consistency checkers, that the information attached to a single edge or node are valid from a physics definition, so independent of interactions. Like checking that the spin magnitude and projection fit together or the Gellmann Nishijima rule.